pre-docs/plan_functions.md

```diff
@@ -345,6 +345,10 @@ get_resources('target1,target2', [Package, File[/etc/puppetlabs]])
 
 Parses common ways of referring to targets and returns an array of Targets.
 
+**NOTE:** When using get_targets inside an apply block with a v2
+inventory, a new Target object will be created. In the case that
+get_targets('all') is called, it will return an empty array.
```

Comment:
Prefer active voice, standardizing on "version 2" instead of "v2", and backticking any words that are actually code:

>**NOTE:** Using `get_targets` inside an `apply` block with a version 2 inventory creates a new Target object. `get_targets('all')` returns an empty array.